automotive/src/main/res/values/strings.xml

```diff
@@ -34,10 +34,10 @@
     <string name="phone_sign_in_button_label">Sign in with phone</string>
     <string name="phone_sign_in_primary_text">Choose phone sign-in method</string>
     <string name="pin_sign_in_button_label">PIN</string>
-    <string name="pin_sign_in_primary_text">Sign in with the PIN code</string>
-    <string name="pin_sign_in_secondary_text">Go to google.com/aural/fast-pair on your phone</string>
+    <string name="pin_sign_in_primary_text">Sign in with PIN code</string>
+    <string name="pin_sign_in_secondary_text">Go to aural.com/fast-pair on your phone</string>
```

Comment:
This URL is completely unrelated to UAMP/Aural, and is owned by an operating company. I don't think we should be pointing to a URL that's impossible to control.

Reply:
changed to example.com/fast-pair